tests/compatibility-tests/pom.xml

```diff
@@ -265,6 +271,39 @@
       </plugins>
    </build>
    <profiles>
+      <profile>
+         <id>compatibility-real-servers</id>
+         <properties>
+            <skipCompatibilityTests>false</skipCompatibilityTests>
+         </properties>
+         <build>
+            <plugins>
+               <plugin>
+                  <groupId>org.apache.activemq</groupId>
+                  <artifactId>artemis-maven-plugin</artifactId>
+                  <version>${project.version}</version>
+                  <executions>
+                     <!-- The executions of dependency-scan will calculate dependencies for each specific version used here on this testsuite. -->
+                     <execution>
+                        <id>snapshot-check</id>
+                        <phase>compile</phase>
+                        <goals>
+                           <goal>dependency-scan</goal>
+                        </goals>
+                        <configuration>
+                           <libList>
+                              <arg>org.apache.activemq:apache-artemis:zip:bin:2.30.0</arg>
+                              <arg>org.apache.activemq:apache-artemis:zip:bin:2.35.0</arg>
```

Comment:
Out of interest, why 2.30.0 and 2.35.0 rather than 2.31.0 (one before the issue was introduced in 2.32.0) and 2.36.0 (the current) ?

Reply:
2.30.0 is one release before the one that changed it.. 2.31.0 would work fine on the roll upgrade.


I can change it to 2.36.0... (I honestly don't know why I chose 2.35.0.. I actually meant the latest and at the time I wrote it, and I guess I didn't think well - confused numbers.)